pkg/kapis/devops/v1alpha3/handler.go

```diff
@@ -291,9 +291,19 @@ func (h *devopsHandler) GetCredential(request *restful.Request, response *restfu
 
 func (h *devopsHandler) ListCredential(request *restful.Request, response *restful.Response) {
 	devops := request.PathParameter("devops")
+
+	// parase filter parameter: keyword
+	conditions, err := params.ParseConditions(request)
+
+	if err != nil {
+		klog.Error(err)
```

Comment:
Should return the error message to the client
```suggestion
		api.HandleBadRequest(response, request, err)
		return
```